src/main/resources/assets/lambdabettergrass/lang/it_it.json

```diff
@@ -0,0 +1,12 @@
+{
+  "lambdabettergrass.menu.info.1": "La mod usa le resource pack per determinare quali blocchi sono affetti.",
+  "lambdabettergrass.menu.info.2": "Questo significa abilitare o disabilitare Better Grass/Snow sui blocchi",
```

Comment:
would it be better to translate Grass/Snow to Erba/Neve here? I am unsure as this translation does recall the name of the mod (hence the capitalisation)

Reply:
right